docs/modules/ROOT/pages/how-to-guides/Secure your supply chain/proc_inspect_sbom.adoc

```diff
@@ -1,26 +1,56 @@
-= SBOMs
+= Inspecting SBOMs
 
 :toc: left
 :icons: font
-:numbered:
 :source-highlighter: highlightjs
 
-== Software bill of materials
-A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM is a list of all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries might enable specific functionality, facilitate development, or serve some other purpose. 
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:../glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
 
-You can use an SBOM to better understand the composition of your software, identify vulnerabilities, and assess the potential impact of any security issues that might arise. You might also need to provide your customers with an SBOM to comply with cybersecurity regulations. 
+You can use an SBOM to better understand your software’s composition, identify vulnerabilities, and assess the potential impact of any security issues that may arise. You may also need to provide your customers with a component’s SBOM to comply with cybersecurity regulations. 
```

Comment:
IBM SG discourages use of `may`. Although it's often wordier, they discourage us using possessive for technology, so not `software's` 

Also, am I missing something? Why are we using `an` for SBOM? SHouldn't it be `a`? 
What do you think of:

`Use a SBOM to better understand the composition of your software, identify vulnerabilities, and asses the potential impact of any security issues. To comply with cybersecurity regulations, you might need to provide your customer with a SBOM component.` 

Note, though it's not a hard "must" we often are encouraged to lead with the "why" (comply with cybersecurity) then get to the action (provide component).